pyproject.toml

```diff
@@ -138,7 +138,9 @@ legacy-tasks = [
     { include-group = "legacy-github" },
     { include-group = "legacy-notifications" },
     { include-group = "legacy-release" },
+    "binary==1.0.1",
     "debugpy==1.8.2",
+    "rich~=13.7",
```

Comment:
Can you use the latest version instead?

Reply:
Sure
Note that I just set the same version as higher in the file